settings.py

```diff
@@ -28,8 +28,91 @@
 # auto-reload
 RELOAD = True
 
-# The shared secret that was generated when the data product was created.
-SHARED_SECRET = 'very_secret123'
+# The Platform Of Trust public key. THIS IS ONLY FOR TESTING PURPOSES.
+POT_PUBLIC_KEY = '''-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxW3WikbznSzVXyFSiJ11
+sEN8qk8SllDlW81n0K5Q5hqdJsZVF9rx9Ec9WSA6g/nHyo8ksavGoYx2cFd7h5eI
+/dBO3dp285aEpR4H3eoievHsC25dN8BdVBq6m4FOcGKJZ65bG2UIaPGBCxyBWWbN
+dSf/Oz1AZgYLIxJY8WpEcZFSlCyFzosNB0ITV89uy21WHru3MgdlelJ66SalFwLL
+zvWjC3J1WYP2JqWfdUCW7DZejW79n/i8cLvf1j5LJIfpAxlbGQBYn3X0c0BXKkQH
+To2JuYvdWb6BtVesmYzXwkX4WhP8gC3oUE2cDdciXEQRZTowWa/z9RE5ls/ew6ty
+WEBndfq5T0+pnBDJXISxzbe7ZboNH5VEKi4zzDcqJOcqexhZSimd21YHVRhcUhya
++G3JCzb3kxK7dROPSA/i0S+TcGyrHz177HbYT7aKs6zoaJsInfMSirEr2wqhVo4G
+Ra4hCUoqK+Trb26K6jKd98uMQrZL6EigobDGs7qfILCSWGStB7qzax35NqrJy9wz
+4yb3Sknd9ujFOTvPNgaMwmv0PC77dAaP46FG+jPS3M/PKAuP0rw2wjjdJRv8eBEB
+FKDg6EtsfQ70q1q7Nm/B4R/SvFOeQWSzHjBE2ssPmDapfK1CjpXTSys4Vvf4XIov
+fZbnsMS87zXeN0TnvkGOspMCAwEAAQ==
+-----END PUBLIC KEY-----'''
+# Public key used for verifying signatures. THIS IS ONLY FOR TESTING PURPOSES.
+PUBLIC_KEY = '''-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAxW3WikbznSzVXyFSiJ11
+sEN8qk8SllDlW81n0K5Q5hqdJsZVF9rx9Ec9WSA6g/nHyo8ksavGoYx2cFd7h5eI
+/dBO3dp285aEpR4H3eoievHsC25dN8BdVBq6m4FOcGKJZ65bG2UIaPGBCxyBWWbN
+dSf/Oz1AZgYLIxJY8WpEcZFSlCyFzosNB0ITV89uy21WHru3MgdlelJ66SalFwLL
+zvWjC3J1WYP2JqWfdUCW7DZejW79n/i8cLvf1j5LJIfpAxlbGQBYn3X0c0BXKkQH
+To2JuYvdWb6BtVesmYzXwkX4WhP8gC3oUE2cDdciXEQRZTowWa/z9RE5ls/ew6ty
+WEBndfq5T0+pnBDJXISxzbe7ZboNH5VEKi4zzDcqJOcqexhZSimd21YHVRhcUhya
++G3JCzb3kxK7dROPSA/i0S+TcGyrHz177HbYT7aKs6zoaJsInfMSirEr2wqhVo4G
+Ra4hCUoqK+Trb26K6jKd98uMQrZL6EigobDGs7qfILCSWGStB7qzax35NqrJy9wz
+4yb3Sknd9ujFOTvPNgaMwmv0PC77dAaP46FG+jPS3M/PKAuP0rw2wjjdJRv8eBEB
+FKDg6EtsfQ70q1q7Nm/B4R/SvFOeQWSzHjBE2ssPmDapfK1CjpXTSys4Vvf4XIov
+fZbnsMS87zXeN0TnvkGOspMCAwEAAQ==
+-----END PUBLIC KEY-----'''
+
+# Private key used for signing. THIS IS ONLY FOR TESTING PURPOSES.
+PRIVATE_KEY = '''-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I was actually wondering - could we for Travis tests use env variables set in Travis and like a `settings.travis.py`? Would leave less of a chance for people to actually copy these to their own things.

Applies to all 3 variables really.

Reply:
Probably. Could be a good solution.

Reply:
Moved the keys to travis environment variables, and added settings.travis.py